commands/cluster-countkeysinslot.md

```diff
@@ -1,7 +1,8 @@
-Returns the number of keys in the specified Valkey Cluster hash slot. The
-command only queries the local data set, so contacting a node
-that is not serving the specified hash slot will always result in a count of
-zero being returned.
+Returns the number of keys in the specified Valkey Cluster hash slot **in the
+currently selected database**. The command only queries the local data set, 
```

Comment:
I don't think we need to bold the new text. I would clarify than when the default is 1, it captures all of the keys in the slot.

Reply:
@madolson 
Could you clarify what you mean by `when the default is 1, it captures all of the keys in the slot`?